sys/net/gnrc/sock/gnrc_sock.c

```diff
@@ -86,6 +86,7 @@ ssize_t gnrc_sock_recv(gnrc_sock_reg_t *reg, gnrc_pktsnip_t **pkt_out,
             if (msg.content.value == _TIMEOUT_MAGIC) {
                 return -ETIMEDOUT;
             }
+            __attribute__((fallthrough));
```

Comment:
> There seems to be some problem using the comment in an if clause:

you mean `#ifdef`? Mhhhh... @cgundogan @BytesGalore what's your opinion here? Also can you test also `/* Falls through */` exactly? Not sure the regexes are configured as such to catch `/* FALLTHROUGH */`.